src/ui/screens/settings/MainSettingsScreen.tsx

```diff
@@ -38,7 +38,12 @@ export const MainSettingsScreen: FC<MainSettingsScreenProps> = ({ navigation })
           onPress={() => navigation.navigate(SETTINGS_STACK.TEMPERATURE_BREACH)}
           RightComponent={ChevronIcon}
         />
-
+        <SettingsItem
+          label="Sync settings"
```

Comment:
can we use the translation mechanism? When the job to translate finally comes would be good to keep it to a minimum (I know there are lots missing!)

Reply:
👍 .

Reply:
Resolving as issue created (#61).